authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationHDFSPlugin.java

```diff
@@ -679,4 +680,57 @@ public Boolean onMetadataUpdated(MetadataObjectChange... changes) throws Runtime
     }
     return Boolean.TRUE;
   }
+
+  @Override
+  protected String getServiceType() {
+    return "hdfs";
+  }
+
+  @Override
+  protected Map<String, String> getServiceConfigs(Map<String, String> config) {
+    String usernameKey = "username";
+    String usernameVal = "admin";
```

Comment:
Same here.
Make them some final string members of the class?

Reply:
OK.